.github/scripts/create_pr_for_staged_changes.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+set -uxo pipefail # enable debugging, prevent accessing unset env vars, prevent masking pipeline errors to the next command
+
+#docs
+#title              :create_pr_for_staged_changes.sh
+#description        :This script will create a PR for staged changes and detect and close duplicate PRs.
+#author		        :@heitorlessa
+#date               :May 8th 2023
+#version            :0.1
+#usage		        :bash create_pr_for_staged_changes.sh {git_staged_files_or_directories_separated_by_space}
+#notes              :Meant to use in GitHub Actions only. Temporary branch will be named $TEMP_BRANCH_PREFIX-$GITHUB_RUN_ID
+#os_version         :Ubuntu 22.04.2 LTS
+#required_env_vars  :COMMIT_MSG, PR_TITLE, TEMP_BRANCH_PREFIX, GH_TOKEN, GITHUB_RUN_ID, GITHUB_SERVER_URL, GITHUB_REPOSITORY
+#==============================================================================
+
+PR_BODY="This is an automated PR created from the following workflow"
+FILENAME=".github/scripts/$(basename "$0")"
+readonly PR_BODY
+readonly FILENAME
+
+# Sets GitHub Action with error message to ease troubleshooting
+function raise_validation_error() {
+    echo "::error file=${FILENAME}::$1"
+    exit 1
+}
+
+function debug() {
+    echo "::debug::$1"
+}
+
+function notice() {
+    echo "::notice file=${FILENAME}::$1"
+}
+
+function has_required_config() {
+    # Default GitHub Actions Env Vars: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+    debug "Do we have required environment variables?"
+    test -z "${TEMP_BRANCH_PREFIX}" && raise_validation_error "TEMP_BRANCH_PREFIX env must be set to create a PR"
+    test -z "${GH_TOKEN}" && raise_validation_error "GH_TOKEN env must be set for GitHub CLI"
+    test -z "${GITHUB_RUN_ID}" && raise_validation_error "GITHUB_RUN_ID env must be set to trace Workflow Run ID back to PR"
+    test -z "${GITHUB_SERVER_URL}" && raise_validation_error "GITHUB_SERVER_URL env must be set to trace Workflow Run ID back to PR"
+    test -z "${GITHUB_REPOSITORY}" && raise_validation_error "GITHUB_REPOSITORY env must be set to trace Workflow Run ID back to PR"
```

Comment:
Worth it adding the `COMMIT_MSG` and `PR_TITLE` variables in this check?

#required_env_vars  :COMMIT_MSG, PR_TITLE, TEMP_BRANCH_PREFIX, GH_TOKEN, GITHUB_RUN_ID, GITHUB_SERVER_URL, GITHUB_REPOSITORY

Reply:
YES, good catch! Doing it now

Reply:
DONE